native_client/BUILD

```diff
@@ -5,10 +5,18 @@ load("//tensorflow:tensorflow.bzl",
 
 cc_library(
     name = "deepspeech",
-    srcs = ["deepspeech.cc", "alphabet.h"],
+    srcs = ["deepspeech.cc", "alphabet.h", "beam_search.h", "trie_node.h"] +
```

Comment:
nit: could there be any readability easy win here by defining all the lm bits outside and just have something like:
```
srcs = ["deepspeech.cc", "alphabet.h", "beam_search.h", "trie_node.h"] + kenlm_srcs + boost_locale_srcs
```

Reply:
I couldn't find any way to make this work with Bazel.

Reply:
:(